packages/core/src/analytics/__tests__/analytics.test.js

```diff
@@ -776,7 +776,13 @@ describe('analytics', () => {
             type: trackTypes.TRACK,
             event,
             properties,
-            context,
+            context: {
+              ...context,
+              event: {
+                ...context.event,
+                uuid: expect.any(String),
```

Comment:
Is the `expect.any` here correct?

Reply:
yes, since this const is only used on the expected result, we must ensure the integration track spy is called with the `uuid` that analytics will create.

Reply:
Oh you're right, I thought `data` was being used somewhere else as well. Resolving it. Thanks!!